core/src/main/java/com/pierfrancescosoffritti/androidyoutubeplayer/core/player/options/IFramePlayerOptions.kt

```diff
@@ -66,6 +68,24 @@ class IFramePlayerOptions private constructor(private val playerOptions: JSONObj
             return this
         }
 
+        /**
+         * Controls if the video is played automatically.
+         * @param autoplay if set to 1: the player will start automatically. If set to 0: the player will not start automatically
+         */
+        fun autoplay(controls: Int): Builder {
+            addInt(AUTO_PLAY, controls)
+            return this
+        }
+
+        /**
+         * Controls if the video is muted.
```

Comment:
Change to something like: "Controls if the player will be initialized mute or not."